src/modules/action-button/action-button.component.spec.ts

```diff
@@ -0,0 +1,61 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { DebugElement } from '@angular/core';
+import { Router } from '@angular/router';
+import { expect } from '@blackbaud/skyux-builder/runtime/testing/browser';
+import { SkyModule } from '@blackbaud/skyux/dist/core';
+import { StacheGridModule } from '../grid/grid.module';
+
+import { StacheActionButtonComponent } from './action-button.component';
+
+describe('Action Button Component', () => {
+  let component: StacheActionButtonComponent;
+  let fixture: ComponentFixture<StacheActionButtonComponent>;
+  let debugElement: DebugElement;
+  let mockRouter: any;
+      class MockRouter {
+          public navigate = jasmine.createSpy('navigate');
+      }
+
+  beforeEach(() => {
+    mockRouter = new MockRouter();
+
+    TestBed.configureTestingModule({
+      declarations: [
+        StacheActionButtonComponent
+      ],
+      imports: [
+        SkyModule,
+        StacheGridModule
+      ],
+      providers: [{ provide: Router, useValue: mockRouter }]
+    })
+    .compileComponents();
+    fixture = TestBed.createComponent(StacheActionButtonComponent);
+    component = fixture.componentInstance;
+    debugElement = fixture.debugElement;
+  });
+
+  it('Should create an action button component.', () => {
+    expect(component).toExist();
+  });
+
+  describe('navigate', () => {
+    it('Should navigate to the new link.', () => {
```

Comment:
Lower-case "should", please. Create a sentence, "StacheActionButtonComponent should navigate to the new link."